packages/lit-dev-content/src/components/litdev-code-language-switch.ts

```diff
@@ -93,33 +117,20 @@ export class LitDevCodeLanguageSwitch extends LitElement {
     const mode = getCodeLanguagePreference();
     return html`
       <button
-        title="Display code as JavaScript"
-        aria-label="Display code as JavaScript"
-        ?disabled=${mode === 'js'}
-        @click=${this._onClickJs}
-      >
-        JS
-      </button>
-
-      <button
-        title="Display code as TypeScript"
-        aria-label="Display code as TypeScript"
-        ?disabled=${mode === 'ts'}
-        @click=${this._onClickTs}
+        role="switch"
+        aria-checked=${mode == 'ts' ? 'true' : 'false'}
+        aria-title="Toggle TypeScript"
+        title=${mode === 'ts' ? 'Disable TypeScript' : 'Enable TypeScript'}
```

Comment:
No action required nit: Should the titles be more along the lines of `Toggle between TypeScript and JavaScript`, rather than enabling/disabling TypeScript?

Reply:
> Chrome suggests a slightly larger contrast between the white text and background:

Fixed!

Reply:
> No action required nit: Should the titles be more along the lines of `Toggle between TypeScript and JavaScript`, rather than enabling/disabling TypeScript?

Good question, not 100% sure about this but ... the reason I used "Toggle TypeScript" for the `aria-label` is that it seemed like `role=switch` was the best role to use, which means that the state of the button is expressed in the `aria-checked` attribute. It seemed like "Toggle between TypeScript and JavaScript" would be confusing in that case, because it's not clear whether TypeScript or JavaScript maps to true vs false. When using voiceover, this reads as "Toggle TypeScript" and then "on"/"off" as you switch between them. 

And for `title`, it also seemed good to make it the same boolean choice for visual users, too.